dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -198,4 +213,42 @@ private static File findLocalRepository() {
     return managedDependencies;
   }
 
+  /**
+   * Configures Maven repositories to search for dependencies.
+   *
+   * @param addMavenCentral if true, add Maven Central to the end of the repository list
+   */
+  public static void configureMavenRepositories(Iterable<String> mavenRepositoryUrls,
```

Comment:
I don't think we're likely to be looking at a non-Maven repository, so perhaps just configureRepositories(urls)


Also, this sounds like you're configuring the repositories themselves, which we're not. So maybe `setRepositories`